packages/ui-link/src/Link/index.js

```diff
@@ -77,6 +77,15 @@ class Link extends Component {
      * The element type to render as (will default to `<a>` if href is provided)
      */
     as: PropTypes.elementType,
+    /**
+     * The ARIA role of the element.
```

Comment:
I'd mention here that the value defined here could be overwritten by `forceButtonRole`